### ISSUE TITLE
Temperature range is hard to read

### ISSUE THREAD

**Issue by dragozet** [chubin/wttr.in#257]
Looks like negative temperature values (Celsius) are hard to read due to the line being all the same colour and no spacing between the temps;

![image](https://user-images.githubusercontent.com/39998307/50918576-9ac1ca80-140e-11e9-9065-32a85d5d8980.png)

Could it be changed to have a space between the temperature range; -6  -  -1  and maybe colour it cyan?
Or use a slash "/"?  -6/-1

**Comment by daarro** [comment]
Totally agree, it should be easily readable at a glance. Also it would be nice to put some clarification about this in the docs.
I'm still not absolutely sure how to interpret it. First, I thought it's just a possible temp range, but a range of about 7-8 degrees difference in a specific time of the day seems too much. So I *suppose* it's sensed temperature vs real one. Am I right?

**Comment by chubin** [comment]
Serhiy ([here](https://twitter.com/pavse/status/1097781995579588608)) suggests using `..` for the intervals, and an explicit `+` if the values are around zero (`-X..+Y` instead of `-X-Y` as it is now):
```
-2..+1 
```

What do you think?
I think that the explicit `+` is a good thing without a doubt; not so sure about `..`, but it is probably a good thing too.

**Comment by pavse** [comment]
I agree with Serhiy because that is me ) Let's see what @dragozet and @daarro say

**Comment by dragozet** [comment]
So here are the options:

(1) explicit `+` with `..`     `-2..+1`
(2) no explicit with `..`     `-2..1`
(3) explicit `+` with `/`     `-2/+1`
(4) no explicit with `/`     `-2/1`

The explicit `+` does not matter to me, I'm ok either way.  It's a regional/cultural preference.  But I do think that the `/` slash is easier to read than `..`? (3 & 4)

Also, the temperature is a range.  So the top left, or current display should show current temperature and not a range.  The range should be kept for the morning/noon/evening/night reports only.

**Comment by daarro** [comment]
Good options. (1) and (2) seems better to me, cause `..` is emphasising more, that it is actually a range.

One other option I stumbled upon when looking around a bit, would be to use "to", so like this:
3 to 6 °C
-1 to 3 °C
-6 to -1 °C
Looks a bit easier on the eyes, however there would be additional overhead of getting different language versions done.

> Also, the temperature is a range. So the top left, or current display should show current temperature > and not a range. The range should be kept for the morning/noon/evening/night reports only.
I agree with this.

**Comment by chubin** [comment]
Let us try it with `..`. In my opinion, it looks much better now:

![screenshot_2019-02-19_20-09-40](https://user-images.githubusercontent.com/3875145/53040760-7c75d280-3482-11e9-9a19-d7b111c09917.png)

Now, if the left number is negative and the right one is positive, a plus sign is shown (it is not show if both numbers are positive).

If both numbers are negative, it looks pretty nice too:

![screenshot_2019-02-19_20-10-13](https://user-images.githubusercontent.com/3875145/53040900-e55d4a80-3482-11e9-8d12-1655cfc2f3da.png)

What do you think?

**Comment by pavse** [comment]
Now it looks really good!

**Comment by daarro** [comment]
Nice!

**Comment by dragozet** [comment]
Much easier to read now.

The current temperature (Top Left) should not be a range, should be current temperature.

**Comment by chubin** [comment]
@dragozet Actually, it is "actual temperature"/"feel like temperature". But if we would write `-2 (+2)`, it would be to hard to understand

**Comment by dragozet** [comment]
Ooohh...  I've been missreading that then!

So for your example above Kiev.  The top left `-2 .. +2` means it's `-2` but FEELS like `+2` and then in the noon forecast box underneath it says `-2 .. +2` means that's the RANGE of a low of `-2` and a high of `+2` for noon.

And for the Ottawa example, `-18 .. -11` means it's `-18` but FEELS like `-11` and then the noon forecast is `-16 .. -12`  which is a RANGE of a low of `-16` and a high of `-12`

Yes, that still needs a bit of UX testing.  I'm guessing if we replace the `..` with `Feels like` you need to deal with all the language translations? Would there be any other symbol or character that you could use instead of `..` since that causes the most confusion between "actual temperature".."feel like temperature" and "Low".."High"

**Comment by pavse** [comment]
Now it does look as a range, because of two dots (and i was sure it's a range): `2..6 °C` 
IMHO `+2 (+6) °C` would much better... What about using of word 'as' inside brackets? For example
`+2 (as +6) °C` eng
`+2 (как +6) °C` rus
`+2 (як +6) °C` ukr
Not sure about other languages though..

**Comment by chubin** [comment]
Actually, everything is even much worse and more complicated: the problem is that depending on the data source backend, we may have sometimes real/fee-like temperature and sometimes ranges. It may also happen that for current time it is real/feel-like and ranges for forecasts. We want to make the backends configurable, because many users are not satisfied with our default datasource and they would prefer something else. We could display it as a range when it is a range, and `XX (YY)` when it is real/feel like.

**Comment by dragozet** [comment]
May `XX (YY)` would be better for current temp/feel.  Or omitting feel all together.  Also I think it might be backwards as of right now.

If I type 'wttr.in/?1nF' I get current temp of -9C and feels like -3C.  Should that be switched?

![image](https://user-images.githubusercontent.com/39998307/53099979-ee86ff80-34f4-11e9-9487-fd81ac0a3da9.png)

If I type `wttr.in/?format=3` I get current temp only of -3C

![image](https://user-images.githubusercontent.com/39998307/53100050-124a4580-34f5-11e9-8d1e-43be98146d1b.png)

**Comment by woo37830** [comment]
I'm using curl wttr.in to display the first 7 lines in a geek_script box against my background.  The difficulty is that the temp shows as 71..73 in yellow and the wind speed in green.  These are hard to read depending upon the desktop on which they appear.  Is it possible to make default be black?  and possibly get rid of the funny graphic (although that's not absolutely necessary )

**Comment by chubin** [comment]
@woo37830 If you append the `T` option (`curl wttr.in/?T`), the colors will be switched off. Is it not that what you need? And also you can try `I` for inverted colors

**Comment by woo37830** [comment]
EXCELLENT!  Thanks for the quick response.  I didn't see that in the  documentation somehow.
